contents/handbook/content/docs-style-guide.mdx

```diff
@@ -138,11 +149,108 @@ This creates the following component in the docs, where the final link will dive
 
 This component is globally available and doesn't need to be imported per page, but doesn't automatically run the query with Max AI (currently).
 
-### Record videos
+### Steps
 
-For flows with many steps, it's often more helpful to include a video instead of a screenshot. We use [Screen Studio](https://screen.studio/) to record videos. Feel free to buy yourself a license.
+Use the `<Steps>` component for content that walks the reader through a strict sequence of instructions. Think how-to guides or step-by-step tutorials. 
 
-Use the following settings:
-- <PrivateLink url="https://posthog.slack.com/archives/C01FHN8DNN6/p1729759474007969">Preset</PrivateLink>
-- Remove any zooming-in for clicks, as this can sometimes make videos hard to follow.
-- For exporting: use MP4, 720, 60 fps, and "web" quality
+<Steps>
+
+<Step title="Install the SDK" badge="required">
+
+Steps are automatically numbered.
+
+</Step>
+
+<Step title="Call the capture method" badge="required">
+
+Write the *content* in **markdown**.
+
+</Step>
+
+<Step checkpoint title="Check for events in PostHog" subtitle="Log in to your PostHog account" badge="optional">
+
+You can also use components <TeamMember name="Edwin Lim" photo /> <TeamMember name="Vincent Ge" photo />
```

Comment:
I think we should replace another step to talk about components. Really important to talk about frequent checkpoints to help readers know if they're on the right track.